tests/networking/tests/networking_multus_links.go

```diff
@@ -0,0 +1,257 @@
+package tests
+
+import (
+	"os"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalhelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/globalparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/networking/nethelper"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/networking/netparameters"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/execute"
+	"github.com/test-network-function/cnfcert-tests-verification/tests/utils/namespaces"
+)
+
+var _ = Describe("Networking custom namespace, ", func() {
```

Comment:
did you mean 
Describe("Networking custom namespace, ", func() - > Describe("Networking custom namespace", func() ?
just deleted the comma

Reply:
No, I can not remove the comma because this is the exact name from the test plan.

Reply:
Ok sure 👍